src/mailadm/app.py

```diff
@@ -3,22 +3,37 @@
 """
 import sys
 import os
-
+import deltachat
 from .web import create_app_from_db_path
 import time
 import threading
 from .db import get_db_path, DB
-from mailadm.commands import prune
+from mailadm.commands import prune, warn_expiring_users
 from mailadm.bot import main as run_bot
 from mailadm.bot import get_admbot_db_path
 
 
 def prune_loop():
     db = DB(get_db_path())
+    ac = deltachat.Account(get_admbot_db_path())
```

Comment:
as discussed it's be better to re-use the existing bot/account object even if it re4quires the bot must first be initialized before warnings will be send.